src/misc/TranslationKey.ts

```diff
@@ -1583,3 +1583,5 @@ export type TranslationKeyType =
 	| "yourMessage_label"
 	| "you_label"
 	| "emptyString_msg"
+	// TODO: add to phrase
```

Comment:
noting to do that before merge

Reply:
Yeah, there also needs to be the German translation. Could someone translate and push it up? or send it to me and I can add it.